lib/cdo/aws/cloudfront.rb

```diff
@@ -31,7 +31,7 @@ class CloudFront
     #   If not provided, the default *.cloudfront.net SSL certificate is used.
     cloudfront_config = {
       pegasus: {
-        aliases: [CDO.pegasus_hostname] + CDO.partners.map {|x| CDO.canonical_hostname("#{x}.code.org")},
+        aliases: [CDO.pegasus_hostname, CDO.advocacy_hostname] + CDO.partners.map {|x| CDO.canonical_hostname("#{x}.code.org")},
```

Comment:
Note that due to https://github.com/code-dot-org/code-dot-org/pull/15058#issuecomment-302728034 not fully migrated I'll need to make this change to the production distribution manually when this gets deployed, so please ping me when this goes out.

Reply:
Thanks @wjordan.  Note that for now, we *only* want this on localhost & staging.  Test & production will come later after we've revised based on feedback.